tools/generate_licenses.py

```diff
@@ -304,11 +304,27 @@ def _add_licenses_manually(licenses: list[_License]) -> None:
     ]
 
 
+def _patch_licenses_manually(licenses: list[_License]) -> None:
+    """手動でライセンス情報を修正する。"""
+    for license in licenses:
+        if license.package_name == "kanalizer":
+            # kanalizerのwheelをビルドするときに使ったライブラリの情報を追加する
+            for p in importlib.metadata.files("kanalizer"):  # type: ignore
+                if p.name == "NOTICE.md":
+                    notice_md = Path(p.locate())
+                    break
+            else:
+                raise Exception("kanalizerのNOTICE.mdが見つかりませんでした。")
+            license.license_text += "\n\n"
+            license.license_text += notice_md.read_text(encoding="utf8")
```

Comment:
encodingの指定は、utf-8としている箇所が多そうなので揃えた方が良さそうです。
```suggestion
            license.license_text += "\n\n"
            license.license_text += notice_md.read_text(encoding="utf-8")
```